docs/Server-Methods.md

```diff
@@ -25,6 +25,17 @@ fastify.ready().then(() => {
 })
 ```
 
+<a name="after"></a>
+### after
+Invoked when all plugins and routes have been loaded
+and executed before the method ``` fastify.ready ```
```

Comment:
This is partially correct. The callback passe to `after(cb)` will be invoked when the current plugin and all the plugin that has been registered under him have finished loading. At the top level, this is right before ready.

(also, remove the space and use single backticks if the code is inline)

Reply:
@mcollina  is more explicit I say "The callback pass to after (cb)" or just "Invoked when the current plugin and all ..." which is more clear?